lib/matplotlib/axes/_base.py

```diff
@@ -916,6 +918,7 @@ def cla(self):
         self.tables = []
         self.artists = []
         self.images = []
+        self.mouseover_set = set()
```

Comment:
I'm not a huge fan of the name. Is this really "artists who have mouseover enabled"? Can we come up with a better name?